src/main/java/fi/aalto/cs/apluscourses/presentation/base/TreeViewModel.java

```diff
@@ -1,9 +1,9 @@
 package fi.aalto.cs.apluscourses.presentation.base;
 
 import java.util.List;
-import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.NotNull;
 
 public interface TreeViewModel {
-  @Nullable
-  List<? extends TreeViewModel> getSubtrees();
+  @NotNull
+  List<? extends TreeViewModel> getChildren();
```

Comment:
without going deeper into the details why `@Nullable` -> `@NotNull`?

Reply:
Previously leaves returned `null`. To streamline things, they now return an empty list.